### PR TITLE
Add some metrics for worker and PD client

### PR DESCRIPTION
**Pull request by huachaohuang**
- tikv_worker_pending_task_total
- tikv_worker_handled_task_total
- tikv_pd_request_total
- tikv_pd_request_duration_seconds

### CODE REVIEW

File: src/pd/client.rs [hunk 1]
```diff
@@ -79,6 +95,10 @@ impl PdClient for RpcClient {
     }
 
     fn bootstrap_cluster(&self, stores: metapb::Store, region: metapb::Region) -> Result<()> {
+        let label = "bootstrap_cluster";
+        let timer = Instant::now();
+        PD_REQUEST_COUNTER_VEC.with_label_values(&[label]).inc();
```
Comment (siddontang) [inline src/pd/client.rs]: The histogram has a counter so we don't need another counter metrics. We can use the counter metrics for region heartbeat especially. 

Btw, I prefer not checking the response header, only recording whole call time is fine.

File: src/util/worker/future.rs [hunk 2]
```diff
@@ -49,6 +50,9 @@ impl<T: Display> Scheduler<T> {
         if let Err(err) = self.sender.unbounded_send(Some(task)) {
             return Err(Stopped(err.into_inner().unwrap()));
         }
+        WORKER_PENDING_TASKS_VEC
```
Comment (siddontang) [inline src/util/worker/future.rs]: better use another PR.

File: src/pd/client.rs (hunk 1 above)
Comment (huachaohuang) [reply src/pd/client.rs]: They're different. The counter shows how many requests are initiated, the histogram shows how many requests are responded and their durations.

**Comment by huachaohuang** [comment]
I will split it into two PRs.

File: src/pd/client.rs (hunk 1 above)
Comment (siddontang) [reply src/pd/client.rs]: If we don't care the result, the histogram has a counter.

**Pull request closed without merging**